packages/twenty-server/src/engine/metadata-modules/workspace-migration/workspace-migration.entity.ts

```diff
@@ -95,6 +95,10 @@ export type WorkspaceMigrationColumnAction = {
   | WorkspaceMigrationCreateComment
 );
 
+export type WorkspaceMigrationAlterForeignTableAlteration =
```

Comment:
not a fan of this name and I don't think you need this type anyway. I don't think we should add `foreignTableAlterations`. You can use the existing `column` field. And you simply ensure the action is either Drop, either Create